EIPS/eip-3156.md

```diff
@@ -87,9 +87,12 @@ function flashLoan(
     IERC20 token,
     uint256 amount,
     bytes calldata data
-) external {
+) external returns (bool) {
```

Comment:
All returned values except `0` will map to `true`.  Recommend instead requiring a sentinel `uint256` or `bytes32` value be returned like `keccak256('3156')` or `0xdeadbeef...` to ensure that a fallback function that just returns "success" or any other positive value won't accidentally overlap with the return value we expect to ensure that the loan was *actually* understood/used.

Reply:
After some consideration of possible sentinel values, I think that using `keccak256("ERC3156FlashBorrower.onFlashLoan")` is good enough.